### PR TITLE
Expand privacy section: existing vectors vs NetInfo

### PR DESCRIPTION
**Pull request by igrigorik**
Context: https://github.com/w3c/netinfo/issues/26.

### CODE REVIEW

**Comment by yoavweiss** [comment]
LGTM

File: index.html [hunk 1]
```diff
@@ -242,11 +242,23 @@
...
+  However, above considerations are not new, and sufficiently motivated attackers may already obtain and exploit such information using existing technologies:
+
+  <ul>
+    <li>The attacker can observe the duration (e.g. time from start of fetch to onload event) of any network fetch, and may get more detailed timing data about the same fetch via the Resource Timing API. Knowing the latency and throughput of the fetch allows the attacker to infer the user's connection type.</li>
+    <li>By repeating the above process the attacker can observe fluctuations in latency and throughput and infer network transitions.</li>
+    <li>By fetching resources from own server the attacker can observe client's IP address, which may change whenever user transitions between networks, allowing them to infer both user's connection type (e.g. certain IP ranges map to certain networks) and behavioral patterns.</li>
+    <li>The attacker can use WebRTC to identify client's public and private IP addresses via STUN, or similar mechanisms.</li>
```
Comment (yoavweiss) [inline index.html:257]: Wow, didn't know that

**Comment by igrigorik** [comment]
@sleevi updated in https://github.com/w3c/netinfo/commit/2d6759d15f3f997bb0537f4920047ce30aa6a587 - getting warmer?

**Comment by sleevi** [comment]
LGTM! :)

**Comment by igrigorik** [comment]
Great! Fixed a few spelling mistakes, squashed, and merging. Thanks for your help with this one!

**Pull request merged**